### PR TITLE
Check for valid Page Load Strategies and then error instead of just a…

### LINKED ISSUE #634: Deserialisation of pageLoadStrategy not defined
In chapter [7. Capabilities](https://w3c.github.io/webdriver/webdriver-spec.html#dfn-user-prompt-handler#h-capabilities), section [7.3 Processing Capabilities](https://w3c.github.io/webdriver/webdriver-spec.html#dfn-user-prompt-handler#h-processing-capabilities):
> Otherwise, set the "pageLoadStrategy" entry in matched capabilities to capability value. 

We need to define a deserialisation of the `pageLoadStrategy`. As it is currently, if the user passes in a string value for the capability it is unconditionally set.

I imagine we need a deserialisation definition of the definition **page load strategy** to looks in the **table of page load strategies** to see if an entry with the _keyword_ of the string exists.

### PR DESCRIPTION
**Pull request by AutomatedTester**
… string. Fixes #634

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/733)

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -2191,7 +2191,7 @@
        in <var>matched capabilities</var> to <var>capability value</var>.
 
      <dt>"<code>pageLoadStrategy</code>"
-     <dd><p>If <var>capability value</var> is not a string,
+     <dd><p>If <var>capability value</var> is not in the <a>page load strategies table</a>,
```
Comment (shs96c) [inline webdriver-spec.html]: Why not just `in the <a>table of page load strategies</a>`. That would be equally good English and avoid the need for the alternative definition below.

Comment (andreastt) [inline webdriver-spec.html]: This also needs to highlight which column to look at in said table. Building on what Simon said above, I would suggest the following wording:

> If capability value is not a keyword in the table of page load strategies,

**Comment by AutomatedTester** [comment]
fixed as requested

**Comment by shs96c** [comment]
I see you've responded to @andreastt too. Merging

**Pull request merged**